frappe/email/doctype/email_account/email_account.py

```diff
@@ -90,6 +90,19 @@ def validate(self):
 			if self.append_to not in valid_doctypes:
 				frappe.throw(_("Append To can be one of {0}").format(comma_or(valid_doctypes)))
 
+	def before_save(self):
+		messages = []
+		as_list = 1
+		if not self.enable_incoming and self.default_incoming:
+			self.default_incoming = False
+			messages.append(_("<b>Default Incoming</b> Unchecked since <b>Enable Incoming</b> was Unchecked"))
```

Comment:
```suggestion
			messages.append(_("{} has been disabled. It can only be enabled if {0} is checked.")
				  .format(
				  	frappe.bold(_('Default Incoming')),
				  	frappe.bold(_('Enable Incoming'))
				  )
			)
```